lib/notification-issue.coffee

```diff
@@ -190,6 +191,9 @@ class NotificationIssue
           return packName unless /^\.\./.test(relativePath)
       @getPackageNameFromFilePath(filePath)
 
+    packageName = /Failed to (load|activate) (the|a) (.*) package/.exec(message)?[3]
```

Comment:
And here: https://github.com/atom/atom/blob/666bb344647d1f96df93c875e404e32bd6c1d419/src/package-manager.coffee#L470 (and maybe in other places)

For reference, grim supports a packageName in the metadata: https://github.com/atom/grim/blob/82c45ef794ce93e4455c29365441c62bf94b90a8/spec/grim-spec.coffee#L108